### PR TITLE
Fix crecto controller spec

### PR DESCRIPTION
**Pull request by mofumofu3n**
### Description of the Change

This PR fixes wrong generated spec by `amber g scaffold` when Crecto is used.

### Alternate Designs

No

### Benefits

Fixes Crecto controller spec.

### Possible Drawbacks

No

### CODE REVIEW

**Comment by faustinoaq** [comment]
@mofumofu3n Nice, Thank you!

I was working on this. Well, I guess you were faster :sweat_smile: :+1:

**Comment by mofumofu3n** [comment]
You were creating the related issue! (#817)
I think it will fix after this issue merged.

File: src/amber/cli/templates/scaffold/controller/crecto/spec/controllers/{{name}}_controller_spec.cr.ecr [hunk 1]
```diff
@@ -36,18 +36,21 @@ class <%= class_name %>ControllerTest < GarnetSpec::Controller::Test
 end
 
 describe <%= class_name %>ControllerTest do
+
+  Spec.before_each do
+    Repo.delete_all(<%= class_name %>)
+  end
+
```
Comment (robacarp) [inline src/amber/cli/templates/scaffold/controller/crecto/spec/controllers/{{name}}_controller_spec.cr.ecr]: This is probably not going to do what you expect. 

> Instructs the spec runner to execute the given block before each spec, regardless of where this method is invoked.
https://crystal-lang.org/api/0.24.2/Spec.html#before_each%28%26block%29-class-method

Due to this weakness with how Spec#before_each is implemented, it should be avoided in generated code at almost all costs.

Comment (mofumofu3n) [reply src/amber/cli/templates/scaffold/controller/crecto/spec/controllers/{{name}}_controller_spec.cr.ecr]: I understand this weakness!

**Comment by robacarp** [comment]
@mofumofu3n thanks!

**Pull request merged**